src/Worker.Extensions.DurableTask/HttpRetryOptions.cs

```diff
@@ -0,0 +1,113 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Net;
+using DurableTaskCore = DurableTask.Core;
+
+namespace Microsoft.Azure.Functions.Worker;
+
+/// <summary>
+/// Defines retry policies that can be passed as parameters to various operations.
+/// </summary>
+public class HttpRetryOptions
+{
+    private readonly DurableTaskCore.RetryOptions coreRetryOptions;
```

Comment:
This is not necessary/used in the worker, so it can be removed. This type can be simplified to just have the properties/defaults set directly with the expected defaults.